docs/targets/logz.md

```diff
@@ -0,0 +1,47 @@
+# Event Target for Logz.io 
+This event target integrates with Logz.io, using received CloudEvent's to ship messages to [Logz.io](https://logz.io/)
+
+## Prerequisites
+* Logz.io account.
+* A Logz.io Shipping [token](https://docs.logz.io/user-guide/tokens/)
+
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **Shipping Token** The API Shipping Token. 
```

Comment:
I am not a fan of the word "the" on both of these bulleted points. Since these are not complete sentences I think it can be removed. I also would remove the ending periods since these are not complete sentences.